raster-foundry/raster-foundry
# Add access control rules for public objects

## Overview

This PR (eventually) creates access control rules for all of the primary authorization objects.

### Checklist

- Tables:
  - [x] scenes
  - [x] datasources
  - [x] shapes
  - [x] workspaces
  - [x] tools (templates)
  - [x] tool runs (analyses)

## Notes

Shapes don't have a visibility, so they're all private right now, so they don't need to be included in the
migration. Workspaces don't exist yet. Things with private/organization visibility don't need to be
included in the migration, since the owner filter will result in authorization for those. Seems like we
overthought this one a little bit.

## Testing Instructions

 * run the migration
 * confirm you can see public scsenes, and that their datasources load
 * confirm you can see public templates
 * confirm the checklist above is entirely included in the PR or has an explanatory note

Closes azavea/raster-foundry-platform#321
